synapse/handlers/space_summary.py

```diff
@@ -200,6 +254,43 @@ async def _summarize_local_room(
             )
         return (room_entry,), events_result
 
+    async def _summarize_remote_room(
+        self,
+        room: "_RoomQueueEntry",
+        suggested_only: bool,
+        max_children: Optional[int],
+        exclude_rooms: Iterable[str],
+    ) -> Tuple[Sequence[JsonDict], Sequence[JsonDict]]:
+        room_id = room.room_id
+        logger.info("Requesting summary for %s via %s", room_id, room.via)
+
+        # we need to make the exclusion list json-serialisable
+        exclude_rooms = list(exclude_rooms)
```

Comment:
Does this mean that `get_space_summary` doesn't actually take a `Sequence[str]` for `exclude_rooms`, but actually needs a `List[str]`?

Reply:
yes. It's annotated as such: https://github.com/matrix-org/synapse/pull/9653/files#diff-3cf3f534d150a0319b7bdd81c884daff978bba8f8c603d7f023d5dd5d73e1ab1R1074